tasks/setup.py

```diff
@@ -8,9 +8,10 @@
 from invoke.tasks import task
 
 from .config import Config, get_full_profile_path, get_local_config
-from .tool import ask, info, is_windows, warn
+from .tool import ask, info, is_windows, warn, is_ci
 
-available_aws_accounts = ["agent-sandbox", "sandbox"]
+
+available_aws_accounts = ["agent-sandbox", "sandbox"] if not is_ci() else ["agent-qa"]
```

Comment:
I would just add `"agent-qa"` to the list, ie, `available_aws_accounts = ["agent-sandbox", "sandbox", "agent-qa"]` as a user may want to run the setup for `"agent-qa"`.